entity-framework/ef6/fundamentals/databinding/wpf.md

```diff
@@ -247,11 +247,11 @@ Add the classes that are defined in the model as data sources for this WPF appli
 
     ![Data Sources](~/ef6/media/datasources.png)
 
--   Select the **Category **data source and drag it on the form.
+-   Select the **Category **data source and drag it on the form.<!-- extra asterisks -->
 
 The following happened when we dragged this source:
 
--   The **categoryViewSource** resource and the** categoryDataGrid** control were added to XAML. For more information about DataViewSources, see http://bea.stollnitz.com/blog/?p=387.
+-   The **categoryViewSource** resource and the** categoryDataGrid** <!-- extra asterisks -->control were added to XAML. For more information about DataViewSources, see http://bea.stollnitz.com/blog/?p=387.<!-- link seems to be broken --> 
```

Comment:
Again, there are no extra asterisks, just misplaced. It should be `**categoryDataGrid**`.

Reply:
Replace the word DataViewSources with CollectionViewSource and the link with https://docs.microsoft.com/dotnet/api/system.windows.data.collectionviewsource.